Dockerfile

```
@@ -0,0 +1,32 @@
+ARG RUBY_IMAGE
+FROM ${RUBY_IMAGE}
+
+ARG VENDOR_PATH
+ARG BUNDLER_VERSION
+
+# For tzdata
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN [ apt-get ] \
+    && ( apt-get update -qq \
+        && apt-get install -qq -y build-essential libpq-dev git tzdata) \
```

Let's add some minor tweaks to reduce the docker image size just a bit:

```suggestion
        && apt-get install -qq -y --no-install-recommends build-essential libpq-dev git tzdata \
        && rm -rf /var/lib/apt/lists/*) \
```